cmd/iam.go

```diff
@@ -1661,6 +1661,36 @@ func (sys *IAMSys) policyDBSet(name, policyName string, userType IAMUserType, is
 	return nil
 }
 
+// PolicyDBGetLDAP is only used by LDAP code, it is similar to PolicyDBGet
+func (sys *IAMSys) PolicyDBGetLDAP(name string, groups ...string) ([]string, error) {
+	if !sys.Initialized() {
+		return nil, errServerNotInitialized
+	}
+
+	if name == "" {
+		return nil, errInvalidArgument
+	}
+
+	sys.store.rlock()
+	defer sys.store.runlock()
+
+	// When looking for a user's policies, we also check if the
+	// user and the groups they are member of are enabled.
```

Comment:
```suggestion
```
We don't support disabling user/group via IAM for LDAP. This comment is not needed.